src/env.cc

```diff
@@ -112,6 +112,28 @@ IsolateData::IsolateData(Isolate* isolate,
 #undef V
 }
 
+void IsolateData::MemoryInfo(MemoryTracker* tracker) const {
+#define V(PropertyName, StringValue)                                           \
+  tracker->TrackField(#PropertyName, PropertyName(isolate()));
+  PER_ISOLATE_SYMBOL_PROPERTIES(V)
+#undef V
+
+#define V(PropertyName, StringValue)                                           \
+  tracker->TrackField(#PropertyName, PropertyName(isolate()));
+  PER_ISOLATE_STRING_PROPERTIES(V)
+#undef V
+
+  tracker->TrackFieldWithSize(
+      "allocator", sizeof(*allocator_), "v8::ArrayBuffer::Allocator");
+  if (node_allocator_ != nullptr) {
+    tracker->TrackFieldWithSize(
+        "node_allocator", sizeof(*node_allocator_), "NodeArrayBufferAllocator");
+  }
```

Comment:
Can you make this an `if ... else`? `node_allocator_` should always refer to the same object as `allocator_` when it’s set